src/components/App/ModalApp.tsx

```diff
@@ -1,25 +1,43 @@
-import { h, Component } from 'preact'
-import { connect } from 'react-redux'
-import { bindActionCreators } from 'redux'
-import EventEmitter from 'eventemitter2'
+import { h, Component, ComponentType } from 'preact'
+import { EventEmitter2 } from 'eventemitter2'
```

Comment:
The original import was
```javascript
import EventEmitter from 'eventemitter2'
```
which is a [backwards-compatibility](https://github.com/EventEmitter2/EventEmitter2/blob/master/lib/eventemitter2.js#L779-L787) in EventEmitter2 implementation